be/src/util/priority_queue.h

```diff
@@ -16,8 +16,16 @@ class PriorityQueue {
     using const_reference = const T&;
     using size_type = typename Container::size_type;
 
+    PriorityQueue() = default;
     ~PriorityQueue() = default;
 
+    // std::array is not movable, so we manually implement the move constructor
+    PriorityQueue(PriorityQueue&& src) noexcept {
```

Comment:
Could you plz implement the move assignment also?

Reply:
Fixed